libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -19,11 +21,32 @@ export const STEP_STACKED_TOP_CLASS = 'fd-wizard__step--stacked-top';
 export const STEP_STACKED_CLASS = 'fd-wizard__step--stacked';
 export const STEP_NO_LABEL_CLASS = 'fd-wizard__step--no-label';
 
+export const WIZARD_CLASS = 'fd-wizard';
+export const WIZARD_NAVIGATION_CLASS = 'fd-wizard__navigation';
+export const WIZARD_CONTENT_CLASS = 'fd-wizard__content';
+export const WIZARD_CONTAINER_WRAPPER_CLASS = 'fd-wizard-container-wrapper';
+export const WIZARD_TALL_CONTENT_CLASS = 'fd-wizard-tall-content';
+export const SHELLBAR_CLASS = 'fd-shellbar';
+export const BAR_FOOTER_CLASS = 'fd-bar--footer';
+
 export const ACTIVE_STEP_STATUS = 'active';
 export const CURRENT_STEP_STATUS = 'current';
 export const UPCOMING_STEP_STATUS = 'upcoming';
 export const COMPLETED_STEP_STATUS = 'completed';
 
+export const HEADER_HEIGHT = 64;
```

Comment:
Will this always be `64`? You can check height by `wizard.querySelector('.' + WIZARD_NAVIGATION_CLASS).clientHeight`